Sources/Parley/Views/TableViewCells/MessageTableViewCell/MessageTableViewCellAppearance.swift

```diff
@@ -46,6 +46,11 @@ public class MessageTableViewCellAppearance: ParleyMessageViewAppearance {
         appearance.balloonContentTextInsets = UIEdgeInsets(top: 12, left: 16, bottom: 12, right: 16)
 
         appearance.messageTextViewAppearance.textColor = UIColor.white
+        appearance.messageTextViewAppearance.linkTintColor =  UIColor(white: 1, alpha: 0.6)
+        
+        appearance.fileIconTintColor = UIColor(white: 1, alpha: 0.6)
+        appearance.fileNameColor = UIColor.white
+        appearance.fileActionColor = UIColor(white: 1, alpha: 0.6)
```

Comment:
Also these are missing for the agent above here?

Reply:
I think these could be removed? Since defaults already set those

Reply:
They are set in default for the agent and are overwritten for the user